tests/ltp/run_ltp_test.sh

```diff
@@ -65,7 +65,7 @@ for file in ${ltp_tests[@]}; do
     # Initialize the variables.
     test_name=${final_test_name#"-ltp-testcases-"}
     ltp_testcase_name=$(echo ${test_name%.*.*} | sed 's/-/\//g; s/_/\//g')
-    test_name="${test_name}-($build_mode)-($test_mode)"
+    test_name="${test_name}-($build_mode)-($test_mode)-($SGXLKL_ETHREADS)"
```

Comment:
This will just a number to the test name. Adding also "threads" suffix will be more clear.

Reply:
Yeah, I agree. I was torn between keeping short test names and still including enough information. I wonder if we can abbreviate "ethreads" somehow. Maybe... "kernel-syscalls-fcntl-fcntl34-(nondebug)-(run-hw)-(et=8)"?

Reply:
I went for the full name, it's clearer.